django/chat/templates/chat/modals/presets/presets_form.html

```diff
@@ -82,52 +88,89 @@
             </div>
           </div>
         {% endif %}
-        <div id="error-message">
-          {% if error_message %}<div class="alert alert-danger" role="alert">{{ error_message }}</div>{% endif %}
-        </div>
-      </form>
+        <div id="error-message"
+             class="alert alert-danger d-none mt-3"
+             role="alert"></div>
+      </div>
     </div>
-  </div>
-  <div class="modal-footer pb-0 pt-1 border-0">
-    <div class="col-12">
-      <div class="row">
-        <div class="col-6">
-          <button id="cancel_preset"
-                  class="btn btn-outline-secondary w-25 flex-fill me-2"
-                  hx-get="{% url 'chat:get_presets' chat_id=chat_id %}"
-                  hx-target="#presets-modal-body"
-                  hx-swap="innerHTML transition:true"
-                  type="button">{% trans "Cancel" %}</button>
- 
-          {% if preset_id %}
-            <button id="delete_preset"
-                    class="btn btn-outline-danger w-25 flex-fill"
-                    hx-get="{% url 'chat:chat_options' chat_id=chat_id action='delete_preset' preset_id=preset_id %}"
+    <div class="modal-footer pb-0 pt-1 border-0">
+      <div class="col-12">
+        <div class="row">
+          <div class="col-6">
+            <button id="cancel_preset"
+                    class="btn btn-outline-secondary w-25 flex-fill me-2"
```

Comment:
Buttons should generally not have specific widths, because the text can overflow depending on screen size. E.g. when the window is tiled to one side, this is what happens:

![image](https://github.com/user-attachments/assets/8f8183d1-978a-4ffb-80c5-bcacd06eb72a)

I think this is a pre-existing problem but it would be easy to fix in this PR - simply get rid of the `w-*` classes on any buttons.